### ISSUE TITLE
Fix S6964 FP: Properties decorated with the [BindNever] attribute

### ISSUE THREAD

**Issue by zsolt-kolbay-sonarsource** [SonarSource/sonar-dotnet#9360]
### Description

The rule should not raise on properties that are decorated with the [BindNever] attribute, as those properties will not be populated during Model Binding.

### Repro steps

```cs
public class Model
{
    [BindNever]
    public int Prop { get; set; } // FP
}
```

### Related information

* C#/VB.NET Plugins version: 9.25.1
* Visual Studio version: 17.10.0
* MSBuild / dotnet version: .NET 8.0
* Operating System: Windows 10
